TombEngine/Objects/TR3/Entity/tr3_monkey.cpp

```diff
@@ -168,11 +168,11 @@ namespace TEN::Entities::Creatures::TR3
 			AI_INFO AI;
 			CreatureAIInfo(item, &AI);
 
-			if (!creature->HurtByLara && creature->Enemy->IsLara())
+			if (!creature->HurtByLara && creature->Enemy != nullptr && creature->Enemy->IsLara())
```

Comment:
I would nest these conditions as a bit of security:
```
if (creature->Enemy != nullptr)
{
    if (!creature->HurtByLara && creature->Enemy->IsLara())
        creature->Enemy = nullptr;
}
```

The second change should probably stay as-is because we'd be wrongly "fixing" the logic otherwise.

Reply:
Had change it :-)